### ISSUE TITLE
Building APK using Buildozer/Kivy

### ISSUE THREAD

**Issue by thrasher-** [kivy/buildozer#459]
When attempting to build an APK (Electrum BTC) file on Ubuntu 16.04 & also Ubuntu 14.04 I get the following error message:

```# Install distribute
# Run 'curl http://python-distribute.org/distribute_setup.py | venv/bin/python'
# Cwd /home/thrasher/Desktop/electrum/.buildozer
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
# Install requirement hostpython2 in virtualenv
# Run 'pip install --target=/home/thrasher/Desktop/electrum/.buildozer/applibs hostpython2'
# Cwd /home/thrasher/Desktop/electrum/.buildozer
Collecting hostpython2
  Could not find a version that satisfies the requirement hostpython2 (from versions: )
No matching distribution found for hostpython2
# Command failed: pip install --target=/home/thrasher/Desktop/electrum/.buildozer/applibs hostpython2
# 
# Buildozer failed to execute the last command
# The error might be hidden in the log above this error
# Please read the full log, and search for it before
# raising an issue with buildozer itself.
# In case of a bug report, please add a full log with log_level = 2
```
Just wondering if anyone has encountered this before?

**Comment by jbillhei** [comment]
See this issue: https://github.com/kivy/python-for-android/issues/620

tl;dr: in your buildozer.spec, add hostpython2 to the requirements option.

**Comment by sassystacks** [comment]
So the error in the #620 was: "AttributeError: 'Context' object has no attribute 'hostpython'". The error shown here is different. I have the same problem after adding hostpython2 to the requirements in the buildozer.spec file I received a similar error. If anyone has a suggestion what might be wrong would really appreciate it. Thanks!

# Install requirement hostpython2 in virtualenv
# Run 'pip install --target=/home/kivy/Desktop/Myprojects/.buildozer/applibs hostpython2'
# Cwd /home/kivy/Desktop/Myprojects/.buildozer
Collecting hostpython2
  Could not find a version that satisfies the requirement hostpython2 (from versions: )
No matching distribution found for hostpython2
# Command failed: pip install --target=/home/kivy/Desktop/Myprojects/.buildozer/applibs hostpython2
# 
# Buildozer failed to execute the last command
# The error might be hidden in the log above this error
# Please read the full log, and search for it before
# raising an issue with buildozer itself.
# In case of a bug report, please add a full log with log_level = 2

**Comment by inclement** [comment]
@sassystacks This issue is unrelated to the one you have posted in. Please post the full log and problem description on the kivy-users mailing list (https://groups.google.com/forum/#!forum/kivy-users).

**Comment by Santiago-R** [comment]
I encountered this same error after a Kivy/Buildozer Virtual Machine reinstall, even with hostpython2 in the requirements.

Turned out I hadn't updated buildozer, as the documentation in the Virtual Machine indicates:

> sudo pip install -U buildozer

**All working well now!**
